index.html

```diff
@@ -0,0 +1,374 @@
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <section class="contact-me filter" id="contact">
+      <div class="info-bar">
+        <p class="info">
+          I'm always interested in hearing about new projects,
+          so if you'd like to chat please get in touch.
+        </p>
+      </div>
+      
+      <form
+        action="https://formspree.io/f/xpzklagg"
+        class="footer-form"
+        method="POST"
+        id="contact-form"
+      >
+        <ul class="form-elements">
+          <li>
+            <input
+              type="text"
+              class="input"
+              name="name"
+              data-id="name"
+              id="full-name"
+              maxlength="30"
+              placeholder="First name"
+              required
+            />
+          </li>
+          <li>
+            <input
+              type="text"
+              class="input"
+              name="name"
+              data-id="name"
+              id="second-name"
+              maxlength="30"
+              placeholder="Last name"
+              required
+            />
+          </li>
```

Comment:
Please make sure that instead of First and Last name in mobile view we can see just the Full Name input.
![image](https://user-images.githubusercontent.com/63932912/147420114-57c3e0e9-28a9-4dc5-bb95-aaba17f18dc0.png)